web/src/message_list_tooltips.ts

```diff
@@ -132,16 +139,19 @@ export function initialize() {
             }
             const $elem = $(instance.reference);
             const local_id = $elem.attr("data-reaction-id");
-            const message_id = rows.get_message_id(instance.reference);
+            assert(local_id !== undefined);
+            // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+            const message_id = rows.get_message_id(instance.reference as HTMLElement);
```

Comment:
The function parameter  type is conflicting with `tippyjs` type upstream. Can we do something else here ?